### PR TITLE
AC Test Fix - LRAC-10825 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10825
[Testray](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1652472822)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase [hunk 1]
```diff
@@ -296,6 +296,14 @@ definition {
...
 			for (var layoutName : list "AC Page 2,AC Page 3") {
 				ACUtils.addPage(layoutName = "${layoutName}");
+
+				Navigator.gotoSitePage(
+					pageName = "${layoutName}",
+					siteName = "Site Name");
+
+				ACUtils.addWidgetToPage(
+					layoutName = "${layoutName}",
+					widgetName = "Blogs");
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase:306]: Hey Vini, why did you add a blog to the page? Is it something related to the data appearing in AC?

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase:306]: Hey @manuferreira, yes. I noticed that when we put a blog or WC or something like that, the page event is processed faster, that way the page appears right on the list and increases the chance of success of the test. I believe that this must occur because the page's `viewPage` event, `viewBlog`, is sent, that is, two events for that same page, so it helps to have a better chance of appearing.

Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase:306]: That makes sense. Thanks for the clarification Vini.

**Review by manuferreira — APPROVED** [review]
Looks good to me. Thank you 👍

**Comment by christianstokes** [comment]
ci:forward
